yo/app/scripts/directives/configuration.js

```diff
@@ -373,12 +375,17 @@ angular.module('dmpApp')
          * Posts a file
          */
         $scope.uploadData = function() {
+
+            // Form scope gets somehow lost.
+            // This is a qnd to retrieve the relevant form value.
+            $scope.delimiter = $('#delimiter').val();
```

Comment:
No idea. Tested around for quiet some time. Scope gets live updates while typing. But when submitting it seams to be using an old context.